qiita_db/test/test_search.py

```diff
@@ -28,7 +28,7 @@ def test_parse_study_search_string(self):
                       "lower(column_name) = lower('altitude') and column_type "
                       "in ('integer', 'float8')")
         exp_samp_sql = ("SELECT r.sample_id,sa.altitude FROM "
-                        "qiita.required_sample_info r JOIN qiita.sample_{0} sa"
+                        "qiita.study_sample r JOIN qiita.sample_{0} sa"
```

Comment:
Very minor but the letter r in this query no longer makes sense, same goes for all the queries you updated in this file, perhaps worth updating it?